web/cypress/integration/api/history.spec.js

```diff
@@ -0,0 +1,7 @@
+describe('History', () => {
+  it('Asserts visiting zone requests history data succesfully', () => {
+    cy.intercept('https://app-backend.electricitymap.org/v4/history?countryCode=FR').as('fetchHistory');
+    cy.visit('/zone/FR?remote=true');
+    cy.wait('@fetchHistory').its('response.statusCode').should('eq', 200);
```

Comment:
Should you also do a check for what is returned? Or are you just looking for a 200 response?